src/modules/date/date-string.transformations.ts

```diff
@@ -0,0 +1,32 @@
+import { Injectable } from '@nestjs/common';
+import { DATE_FORMATS } from '@ukef/constants';
+import { DateOnlyString } from '@ukef/helpers/date-only-string.type';
+import { DateString } from '@ukef/helpers/date-string.type';
+import { matches } from 'class-validator';
+
+@Injectable()
+export class DateStringTransformations {
+  /**
+   * Remove time part from DateTime string.
+   */
+  removeTime(dateTime: string): DateString {
```

Comment:
input can be changed to "date: DateString"
return type can be changed to DateOnlyString